cpp/src/aztec/plonk/proof_system/prover/prover.test.cpp

```diff
@@ -252,11 +243,11 @@ waffle::Prover generate_test_data(const size_t n)
     q_m.ifft(key->small_domain);
     q_c.ifft(key->small_domain);
 
-    polynomial q_1_fft(q_l, n * 2);
-    polynomial q_2_fft(q_r, n * 2);
-    polynomial q_3_fft(q_o, n * 2);
-    polynomial q_m_fft(q_m, n * 2);
-    polynomial q_c_fft(q_c, n * 2);
```

Comment:
I'd say it was a typo/bug yes. Before my changes it would've called bump_memory under the hood but just setting the right value in the first place is obviously preferable.